src/solvers/CMakeLists.txt

```diff
@@ -0,0 +1,85 @@
+#   TODO Specify which solver to use (minisat2 etc.)
```

Comment:
Could you try using glucose and document the changes required to get it to run?

Reply:
I'll do that now.